sim/warlock/immolate.go

```diff
@@ -87,10 +87,10 @@ func (warlock *Warlock) getImmolateConfig(rank int) core.SpellConfig {
 		},
 
 		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
-			imprImmoMult := 1 + 0.05*float64(warlock.Talents.ImprovedImmolate)
-			spell.DamageMultiplier *= imprImmoMult
+			oldMultiplier := spell.DamageMultiplier
+			spell.DamageMultiplier *= 1 + 0.05*float64(warlock.Talents.ImprovedImmolate) // TODO should most likely just be done statically (?)
```

Comment:
Should not apply to the Dot part of the spell. which is why its done like this. If the dot spell is taken out as a different one then we can apply statically to the initial spell